Meadowlark-js/tests/e2e/helpers/Shared.ts

```diff
@@ -3,8 +3,23 @@
 // The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
 // See the LICENSE and NOTICES files in the project root for more information.
 
-import { chance, rootURLRequest } from '../Setup';
-import { Clients, getAccessToken } from './Credentials';
+import Chance from 'chance';
+import memoize from 'fast-memoize';
+import request, { SuperTest, Test } from 'supertest';
+import { getAccessToken } from './Credentials';
+
+const chance = new Chance() as Chance.Chance;
+
+function getBaseURLRequest() {
+  return request(process.env.BASE_URL);
```

Comment:
`BASE_URL` appears to just be the root URL + stage. Could change BASE_URL to stage then, to more closely match fastify settings. But this is working, so probably not worth changing that right now.

Reply:
The only aspect to taken into account when we do this change is that the root URL is used to access resources by Location, because when a new resource is created, we use the location to delete the element after the test. The location path includes the stage, so this could be handled with logic